Documentation/Getting-Started/MaxGUI.md

```diff
@@ -8,4 +8,57 @@ MaxGUI is the new browser based tool for configuring and managing
 MaxScale. It is a more user friendly and intuitive to use companion
 to the command line tool `maxctrl`.
 
-# ...
+# Enabling MaxGUI
+
+In order to make MaxGUI works, the following parameters al least needs to be enabled in MaxScale configuration file:
```

Comment:
Please use hard line breaks at roughly 76 chars.
When the md file is rendered or imported to MariaDB KB, those hard linebreaks will be ignored.
But with very longs lines the diffs become hard to evaluate.